### DISCUSSION TITLE
Made conversion of latest ( 22.01.2023 version) of the plugin to Godot 4 Beta 14.

### DISCUSSION THREAD

**Discussion by Gone2Daly** [fenix-hub/godot-engine.easy-charts#62] [Show and tell]
Well title say it, also somehow on my way deleted ability to dynamically scale graph, probably some thing I don't know yet about Godot doing it. Anyway - good luck to anyone with their projects.

Good plugin fenix, keep it up! If possible - do pie charts next.
[easy_charts.zip](https://github.com/fenix-hub/godot-engine.easy-charts/files/10475070/easy_charts.zip)

**Comment by fenix-hub** [comment]
@Gone2Daly 
Thanks for your effort!
Would you mind making a pull request? I can directly review the code as soon as possible.
Currently I'm not actively porting the plugin to 4.x, but only because I'm waiting for a stable release of Godot 4 (don't want to rewrite everything again for each new beta, since sometimes APIs are renamed or removed), but I could for sure use your effort as a base, since it is already functional.
Let me know if you need help with this, if you are inclined to.

**Comment by Gone2Daly** [comment]
My first pull request >_<, so I think I made one not right. But right now everything should be right.

Well I'm doing my game where charts and stuff needed, so you can count on me converting your stuff for my own purposes, and I will be glad to contribute giving you what I did for beta's.

**Reply by fenix-hub** [reply]
Thank you, and don't worry about making mistakes :)
Happy to know you are using this plugin for a game. Feel free to contribute, or just propose new features you would like to see.

**Comment by spaceoddity-carlos** [comment]
Now that there's a stable Godot 4 release and charts had a new breaking update could we pretty please get a port?

**Reply by fenix-hub** [reply]
Hi @spaceoddity-carlos,
review planned for today!

**Reply by spaceoddity-carlos** [reply]
Do you have Kofi or patreon or something like that? wouldn't mind buying you a coffee or two as encouragement to keep this addon going :)

**Reply by fenix-hub** [reply]
Hi @spaceoddity-carlos , thank you for your interest.
No actually I don't have any of those platforms. I think I enabled github sponsors some time ago and it should be visible here on the easy charts repository.
If you really want to donate something please let me know what platform would you prefer. It would for sure help me :) But I want to stress that you don't have to.

**Reply by spaceoddity-carlos** [reply]
I'll look into github sponsors :)
